app/screens/error_teams_list/error_teams_list.js

```diff
@@ -67,13 +67,17 @@ export default class ErrorTeamsList extends PureComponent {
     };
 
     getUserInfo = async () => {
-        this.setState({loading: true});
-        this.props.actions.connection(true);
-        await this.props.actions.loadMe();
-        this.props.actions.connection(false);
-        this.setState({loading: false});
-        this.props.actions.selectDefaultTeam();
-        this.goToChannelView();
+        try {
+            this.setState({loading: true});
+            this.props.actions.connection(true);
```

Comment:
On a different note: Why do we need to stop making requests if the flag is set to false?

Reply:
if the flag is set to false then we stop making the requests https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.js#L2487 and I believe this was cause of some issues in Android that the object was kept opened and caused a OOM